api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirPersonServiceImpl.java

```diff
@@ -29,18 +31,17 @@
 @Component
 @Transactional
 @Setter(AccessLevel.PACKAGE)
-public class FhirPersonServiceImpl implements FhirPersonService {
+public class FhirPersonServiceImpl extends BaseFhirService<Person, org.openmrs.Person> implements FhirPersonService {
 	
 	@Autowired
 	private FhirPersonDao fhirPersonDao;
 	
 	@Autowired
 	private PersonTranslator personTranslator;
 	
-	@Override
 	@Transactional(readOnly = true)
 	public Person getPersonByUuid(String uuid) {
-		return personTranslator.toFhirResource(fhirPersonDao.getPersonByUuid(uuid));
+		return personTranslator.toFhirResource(fhirPersonDao.get(uuid));
 	}
```

Comment:
We shouldn't need this method any more.

Reply:
So we should remain as it getPersonByUuid(); not get(uuid). is that what you are meaning

Reply:
No. Just remove the declaration. It's already built into the BaseFhirService class. We also don't want the `getPersonByUuid()` method. Done right, we end up with less code doing this :grin:.